src/webui/api/serialize/serialize_torrent.cpp

```diff
@@ -103,7 +103,7 @@ QVariantMap serialize(const BitTorrent::Torrent &torrent)
     };
 
     return {
-        {KEY_TORRENT_HASH, QString(torrent.hash().toString())},
+        {KEY_TORRENT_ID, QString(torrent.id().toString())},
```

Comment:
Later we would need to send sha1 and sha256 in 2 additional fields.

Reply:
Will add TODO.